src/cljs/rems/administration/create_form.cljs

```diff
@@ -135,8 +135,10 @@
              [language (trim-when-string (get lstr language ""))])))
 
 (defn- string-is-not-empty [s]
-  (when (string? s)
-    (not (str/blank? s))))
+  (if (string? s)
+    (not (str/blank? s))
+    false
+    ))
```

Comment:
as I mentioned earlier, you should be able to just define

```
(defn- string-is-not-empty [s]
  (not (str/blank? s)))
```

Reply:
Also I think it's better to just not define that function and use not blank directly.

Reply:
Why  not define it?

Reply:
`str/blank?` is as good as standard function, no need to define one. Also `not a function` does not need a new name, especially so trivial as this one is. Also a later refactoring could remove the `not` entirely too.